fe/fe-core/src/main/java/com/starrocks/statistic/StatisticUtils.java

```diff
@@ -204,24 +208,29 @@ public static Optional<Double> convertStatisticsToDouble(Type type, String stati
         if (!type.canStatistic()) {
             throw new StarRocksPlannerException("Error statistic type : " + type.toSql(), ErrorType.INTERNAL_ERROR);
         }
-        switch (type.getPrimitiveType()) {
-            case BOOLEAN:
-                if (statistic.equalsIgnoreCase("TRUE")) {
-                    return Optional.of(1D);
-                } else {
-                    return Optional.of(0D);
-                }
-            case DATE:
-                return Optional.of((double) getLongFromDateTime(DateUtils.parseStringWithDefaultHSM(
-                        statistic, DateUtils.DATE_FORMATTER_UNIX)));
-            case DATETIME:
-                return Optional.of((double) getLongFromDateTime(DateUtils.parseStringWithDefaultHSM(
-                        statistic, DateUtils.DATE_TIME_FORMATTER_UNIX)));
-            case CHAR:
-            case VARCHAR:
-                return Optional.empty();
-            default:
-                return Optional.of(Double.parseDouble(statistic));
+        try {
+            switch (type.getPrimitiveType()) {
+                case BOOLEAN:
+                    if (statistic.equalsIgnoreCase("TRUE")) {
+                        return Optional.of(1D);
+                    } else {
+                        return Optional.of(0D);
+                    }
+                case DATE:
+                    return Optional.of((double) getLongFromDateTime(DateUtils.parseStringWithDefaultHSM(
+                            statistic, DateUtils.DATE_FORMATTER_UNIX)));
+                case DATETIME:
+                    return Optional.of((double) getLongFromDateTime(DateUtils.parseStringWithDefaultHSM(
+                            statistic, DateUtils.DATE_TIME_FORMATTER_UNIX)));
+                case CHAR:
+                case VARCHAR:
+                    return Optional.empty();
+                default:
```

Comment:
Can we add a branch to process NULL type here?

Reply:
No, NULL values should not enter this function, and our current implementation logic is that NULL Literal can be of any type, and the type depends on the calculation involved